src/Http/Session.php

```diff
@@ -653,14 +653,12 @@ public function renew(): void
 
         $this->start();
         $params = session_get_cookie_params();
+        unset($params['lifetime']);
+        $params['expires'] = time() - 4200;
         setcookie(
```

Comment:
Please add  `/** @phpstan-ignore-next-line */` above the `setcookie()` to stop phpstan from complaining.